app/views/admin/profile_fields/index.html.erb

```diff
@@ -1,36 +1,20 @@
 <main id="profileFields">
-  <article class="crayons-card grid p-6 mb-6 gap-1">
```

Comment:
I've removed the creating of new fields and groups temporarily. We can currently edit what we have on the interface. The creation of these will require some rework + implementation of the updated designs (which caters for this new model) which I don't believe needs to hold this PR back. https://www.figma.com/file/oo9B9YSe8aWkuLKPBLc0g6/Profile-Generalization?node-id=29%3A718